packages/client/hmi-client/src/components/workflow/ops/compare-simulations/compare-simulations-operation.ts

```diff
@@ -0,0 +1,22 @@
+import { WorkflowOperationTypes } from '@/types/workflow';
+import type { Operation, BaseState } from '@/types/workflow';
+
+export interface CompareSimulationsState extends BaseState {}
+
+export const CompareSimulationsOperation: Operation = {
+	name: WorkflowOperationTypes.COMPARE_SIMULATIONS,
+	displayName: 'Compare simulations',
+	description: 'Compare simulations',
+	documentationUrl: '',
+	inputs: [
+		{ type: 'datasetId', label: 'Dataset' },
+		{ type: 'datasetId', label: 'Dataset' }
+	],
```

Comment:
If we are taking in datasetIds, and "only" datasetIds, I think we are better off just calling this operator CompareDatasets ?? It would be a bit less confusing?? 

Or does this operator only work for dataset derived from a simulation?

Reply:
Yes, let's just call it "Compare Datasets" for now until Jamie/Pascale says otherwise.